### PR TITLE
Update docs based on experience with a customer

### PR DESCRIPTION
**Pull request by duchatran**
## Purpose of this pull request

This pull request update our doc pages for BigQuery and Gmail Trace Logs (which is now not working anymore)

Issue number: CON-1848

## Select the type of change:
- [x] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md [hunk 1]
```diff
@@ -5,10 +5,16 @@ sidebar_label: Gmail Trace Logs
...
 ---
 
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/send-data/gmail-trace-logs-icon.svg')} alt="Gmail" width="50"/>
 
+:::note
+This source was originally developed for [Gmail logs in BigQuery](https://support.google.com/a/topic/7233311?fl=1&sjid=293657922002214011-NA), which is now replaced by [Google Workspace logs and reports in BigQuery](https://support.google.com/a/topic/9079469?fl=1&sjid=293657922002214011-NA).
+Please use our [Google BigQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source), see Example 3, instead
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md:15]:
```suggestion
This source was originally developed for [Gmail logs in BigQuery](https://support.google.com/a/topic/7233311?fl=1&sjid=293657922002214011-NA), which has been replaced by [Google Workspace logs and reports in BigQuery](https://support.google.com/a/topic/9079469?fl=1&sjid=293657922002214011-NA).

Please use our [Google BigQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source) instead.
```

Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md:15]: @duchatran you mentioned `Example 3` - what are you referring to?

With this note, we are redirecting users to BigQuery source. In what scenario(s) should customers continue to use  Gmail Trace Logs source? Or is it now obsolete?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ To configure an Google BigQuery Source:
 1. **Checkpoint Field**. Enter the name of the field in the query result to be used for checkpointing. This field has to be increasing and of type number or timestamp.
 1. **Checkpoint Start**. Enter the first value for the checkpoint that the integration will plug into the query.
 1. **(Optional) Time Field**. Enter the name of the field in the query result to be parsed as timestamp. If not provided, the current time will be used.
-1. **Query**. Enter the query that you need to run. It must include the phrase `%CHECKPOINT%`.
+1. **Query**. Enter the query that you need to run. It must include the phrase `%CHECKPOINT%` and *should* include sorting the checkpoint field as well.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:65]: is this correct?

```suggestion
1. **Query**. Enter the query that you need to run. It must include the phrase `%CHECKPOINT%`, and we recommend including the sorting checkpoint field as well.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 3]
```diff
@@ -116,17 +116,22 @@ SELECT trip_id,subscriber_type,start_time,duration_minutes FROM bigquery-public-
 | `Checkpoint Start`| `0` |
 | `Time Field` | `start_time` |
 
-#### Example 3: Query Gmail Logs
+#### Example 3: Query Gmail Logs 
+(Replace `MyProject` and `MyDataSet` with values matching your environment)
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:120]:
```suggestion
#### Example 3: Query Gmail Logs 

In the example below, you'll need to replace `MyProject` and `MyDataSet` with values matching your environment.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
follow up questions

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md (hunk 1 above)
Comment (duchatran) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md]: Gmail Trace Logs no longer works for customers who setup Gmail logs anew. I am just not sure if it still works for existing customers so decided to leave this page out with this note

Comment (duchatran) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md]: Oh I overlooked at didn't recognize you removed "Example 3". That source is a more generic BigQuery source, and Example 3 is for Gmail specifically.

Comment (duchatran) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md]: (Added it back in the latest commit)

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md [hunk 4]
```diff
@@ -5,10 +5,17 @@ sidebar_label: Gmail Trace Logs
...
 
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/send-data/gmail-trace-logs-icon.svg')} alt="Gmail" width="50"/>
 
+:::note
+This source was originally developed for [Gmail logs in BigQuery](https://support.google.com/a/topic/7233311?fl=1&sjid=293657922002214011-NA), which has been replaced by [Google Workspace logs and reports in BigQuery](https://support.google.com/a/topic/9079469?fl=1&sjid=293657922002214011-NA).
+
+Please use our [Google BigQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source) instead (see Example 3 under [this section](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source/#sample-values-for-query-checkpoint-and-checkpoint-start-fields) specifically).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md:16]:
```suggestion
You'll need to use our [Google BigQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source) instead (see [Example 3: Query Gmail Logs](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source)). 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
couple more questions

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md (hunk 2 above)
Comment (duchatran) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:65]: sounds good!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**